rollup.config.js

```diff
@@ -17,6 +18,7 @@ export default {
   },
   plugins: [
     json(),
+    replace({ __BASE_DOMAIN__: production ? '' : 'http://localhost:8000' }),
```

Comment:
This uses the `ROLLUP_WATCH` from line 9 instead of the one you added in the build script. I think to use that you'd instead do:
```
     replace({ __BASE_DOMAIN__: process.env.NODE_ENV === 'production' ? '' : 'http://localhost:8000' }),
```
??